.github/workflows/geth.yml

```diff
@@ -0,0 +1,13 @@
+name: Geth Interaction
+
+on:
+  pull_request:
+    types: [synchronize, opened, reopened, ready_for_review]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Geth and bus-mapping test
+      uses: actions/checkout@v2
+    - run: sh web3-utils/script/geth.sh
```

Comment:
Ok, I saw now at the end of the review what you're trying to do.

Seems nice then to go on that way but I'll do two things:
1) Tests that are integration tests should be behind a feature flag. So that with the normal `cargo test` command they're not triggered.
2) Rename this workflow to `Integration Tests` or something similar. As `Geth Integration` doesn't really tell what this does.
3) Here we should probably add all of the env variables that have been suggested to be used. Such as `geth_address` and this kind of stuff.